### ISSUE TITLE
Download files from GFonts when they're not found locally

### ISSUE THREAD

**Issue by felipesanches** [googlefonts/FontClassificationTool#14]
The "David Libre" family is available at
https://fonts.google.com/specimen/David+Libre

But not on the google/fonts github repo. For that reason it ends up not having its weight & width values auto-calculated and also no being loaded into the tool for manual selection of italic angle values

Maybe the tool could automatically fetch files from GFonts in cases like that.

**Comment by felipesanches** [comment]
These also seem to be missing from the google/fonts git repo:

* Nanum Gothic:normal:800
* Nanum Myeongjo:normal:800

**Comment by davelab6** [comment]
No, I should have done this (getting them into git already) before. File
issues here and I'll work through them

**Comment by felipesanches** [comment]
ok, cool! Thanks
